rqt_image_view/include/rqt_image_view/image_view.h

```diff
@@ -82,27 +83,31 @@ protected slots:
   virtual QSet<QString> getTopics(const QSet<QString>& message_types, const QSet<QString>& message_sub_types, const QList<QString>& transports);
 
   virtual void selectTopic(const QString& topic);
+  virtual void selectCwRotation(const int rotation);
 
 protected slots:
 
   virtual void onTopicChanged(int index);
+  virtual void onRotationChanged(int index);
 
-  virtual void onZoom1(bool checked);
+  virtual void onZoom(bool checked);
```

Comment:
I still don't see a reason to change the name of this function. If other packages use this API they are needlessly broken by this.